modules/extension/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1846,38 +1851,42 @@ public List<FeatureTypeStyle> transformInternal(MBStyle styleContext) {
                         halo,
                         fill);
         symbolizer.fonts().clear();
-        symbolizer.fonts().addAll(fonts);
+        // no need to add fonts and alternatives if we are displaying a simple symbol
+        if (!pureSymbol) {
+            symbolizer.fonts().addAll(fonts);
+        } else {
+            // tiny font
+            Font font =
```

Comment:
This should be created statically with a comment at the top of the class since it is basically a constant representing the tiny font

Reply:
Yep, done